pkg/note/note.go

```diff
@@ -74,14 +74,17 @@ type noteVerifier struct {
 	verify func(msg, sig []byte) bool
 }
 
+// Name implements note.Verifier.
 func (n *noteVerifier) Name() string {
 	return n.name
 }
 
+// Keyhash implements note.Verifier.
 func (n *noteVerifier) KeyHash() uint32 {
 	return n.hash
 }
 
+// Verify implements note.Verifier.
 func (n *noteVerifier) Verify(msg, sig []byte) bool {
```

Comment:
We should re-use the original docs

 - https://pkg.go.dev/golang.org/x/mod/sumdb/note#Verifier